docs/api/api.md

```diff
@@ -259,16 +300,28 @@ The Mbed OS classes providing USB peripheral functionality, also known as USB co
 </tbody>
 </table>
 
+### RTOS only supported APIs
+<table>
+<tbody>
+<tr>
+<td><a href="usbcdc-ecm.html">USBCDC_ECM</a></td>
+</tr>
+</tbody>
+</table>
+
 ## Security
 
 With [Arm Mbed TLS](security.html), a comprehensive SSL/TLS solution, you can include cryptographic and SSL/TLS capabilities in your code.
 
+### RTOS only supported APIs
 <table>
 <tbody>
 <tr>
 <td><a href="security-apis.html">PSA initial attestation</a></td>
 <td><a href="psa-lifecycle.html">PSA lifecycle</a></td>
 <td><a href="mbed-crypto.html">Mbed Crypto</a></td>
+</tr>
```

Comment:
Crypto is listed as available in bare metal on the main table. Similarly for Device Ket and Mbed TLS.